src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceContractTests.cs

```diff
@@ -336,8 +334,7 @@ public static void NetTcp_NoSecurity_Echo_RoundTrips_String_StreamedResponse()
 
     [Fact]
     [OuterLoop]
-    [ActiveIssue(713)]
-    public static void NetTcp_NoSecurity_Echo_RoundTrips_String_Streamed()
+    public static void NetTcp_NoSecurity_Streamed_RoundTrips_String()
```

Comment:
I'm having 2nd thoughts about [Theory] tests whenever I discover some NET Native or *nix specific issue.  It's like the [Theory] data enumerator itself needs a way to mark certain values as having an ActiveIssue.  I'm open to ideas but will probably leave these separate till I prove it works across all platforms.